TestON/tests/FuncIntent/Dependency/FuncIntentFunction.py

```diff
@@ -10,6 +10,7 @@ def hostIntent( main,
                 name,
                 host1,
                 host2,
+                onosNode=0,
```

Comment:
You may want to typecast this to an int just incase it is passed straight from a params file